testsuite/domain/src/test/resources/host-configs/host-failover2.xml

```diff
@@ -55,7 +55,7 @@
     </management>
 
     <domain-controller>
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm"/>
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false"/>
```

Comment:
It's odd this is needed. DomainControllerMigrationTestCase is the only user of this file, and when it sets up the HCs, for each it calls WildFlyManagedConfiguration.setBackupDC(true). So the xml setting should not be required.